superset-frontend/src/SqlLab/components/TableElement.tsx

```diff
@@ -53,16 +70,21 @@ const StyledSpan = styled.span`
 
 const Fade = styled.div`
   transition: all ${({ theme }) => theme.transitionTiming}s;
-  opacity: ${props => (props.hovered ? 1 : 0)};
+  opacity: ${(props: { hovered: boolean }) => (props.hovered ? 1 : 0)};
 `;
 
-const TableElement = props => {
+// Note about TableElement props:
+// The antd Collapse component is expecting its children to be `Collapse.Panel`s and
+// is quietly passing extra props that need to be on the panels for them to work correctly.
+// These props are not defined in their TypeScript type for Panel though because
+// this logic is happening in the Collapse component itself. We have gotten around the TypeScript
+// errors by using the rest and spread operators to pass the necessary extra props to the Panels.
+
```

Comment:
I think this comment makes sense, but the pattern below doesn't look confusing. I think it's ok to remove the comment.

Reply:
Sounds good! I am removing the comment and adding the two other changes.  I will push them all up momentarily.